doc/source/archiver.rst

```diff
@@ -0,0 +1,122 @@
+.. currentmodule:: databroker.eventsource.archiver
+
+
+**********************************
+Connection with Archiver Appliance
+**********************************
+
+The Archiver Appliance is an EPICS logging system developed in
+a collaboration of SLAC, BNL, and MSU for collecting and
+storing measurements from various control devices. Data retrieval
+is based on the client-server interface using HTTP requests.
+Large-scale accelerators and experimental facilities may maintain
+multiple archivers. Within the databroker application, each archiver
+is represented by an ArchiverEventSource that needs to be configured
+and registred with ArchiverReader. ArchiverEventSource can either be
+integrated with databroker or used as a stand-alone class to address
+different requirements. The following sections highlight four major
+topics: ArchiverEventSource configuration, its integration with databroker,
+data retrival based on the databroker interface, and a stand-alone
+use case.
+
+Configuration
+-------------
+
+:class:`ArchiverEventSource` is configured via :class:`ArchiverReader`
+using a Python dictionary with four key-value pairs :
+
+* name: user-defined name of archiver
+* url: address of the Archiver Appliance Retrieval server
+* timezone: time zone (e.g., 'US/Eastern')
+* pvs: dictionary mapping user-defined names to EPICS PVs
+
+For example, the following code illustrates the configuration of
+two :class:`ArchiverEventSource` instances:
+
+.. code-block:: python
+
+    # CSX ArchiverEventSource
+    aes_config_csx = {
+		'name' : 'csx',
+		'url' : 'http://xf23id-ca.cs.nsls2.local:17668',
```

Comment:
Do we have a list someplace of all of the archiver URLs?  That should probably go into playbooks.